docs/reference/repositories-stats/apis/clear-repositories-stats-archive.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[clear-repositories-stats-archive-api]]
+=== Clear repositories statistics archive API
+++++
+<titleabbrev>Clear repositories statistics archive API</titleabbrev>
+++++
+
+Removes the archived repositories statistics in the cluster.
+
+[[clear-repositories-stats-archive-api-request]]
+==== {api-request-title}
+
+`DELETE /_nodes/<node_id>/_repositories_stats`
```

Comment:
After I had some time to think about this ... I wonder, why make life for users so hard by adding `node_id` here. Wouldn't it be much easier if we just allowed for passing a list of `emphemeral_id` for archived entries instead so that the consuming API can delete after having safely persisted an id avoiding essentially all kinds of possible races and making this much easier to use?
In the implementation we could simply broadcast the delete to all nodes so we don't have to figure out what id lives on what node I guess to keep it simple.

Reply:
@ywelsch maybe you have an opinion on this as well?

Reply:
if we think this API would be hard to use, maybe we can limit its scope to a single node? I added it as I needed a way to clear the archive between tests, if there's a simpler way that can simplify this I'm open to it 👍 .

Reply:
Perhaps we can use a logical timestamp here to say "only clear repositories that I've observed", and use the cluster state version as such a logical timestamp. I think I would prefer that over a list of all IDs for archived entries. The API would still allow the ability to select nodes (in particular allow for _local).